### ISSUE TITLE
Update infrastructure for Hub server

### ISSUE THREAD

**Issue by VikramTiwari** [gdg-x/hub#118]
Hub is currently hosted on an Ubuntu VM with OS version 15.04. This is a deprecated version. During resolution of https://github.com/gdg-x/hub/issues/98 I realized that there is an open security bug which is causing our SSL score to be down even when we are using LetsEncrypt.

For comparison:
[My website score](https://www.ssllabs.com/ssltest/analyze.html?d=vikramtiwari.com) is `A+` while [hub's score](https://www.ssllabs.com/ssltest/analyze.html?d=hub.gdgx.io&latest) is `F`.

My suggestion would be to move project into a new VM with either Debian 8 or Ubuntu 16.10

**Comment by Splaktar** [comment]
Tried to upgrade ubuntu on the same VM today but it failed as unsupported. Looks like we'll need to spin up a new VM.

**Comment by VikramTiwari** [comment]
Yeah! I tried that earlier as well but since it's on a non-LTS version, the upgrade won't make through.

I can take point on this over the weekend. Though if you have some time please do the following:
- A basic documentation/script on all the things that are needed to run the hub project
- Share any env files over gdrive
- Anything else I should know (DNS mapping, static IP addresses etc)

Let me know what you think.

**Comment by Splaktar** [comment]
I tried to document everything here https://github.com/gdg-x/hub/wiki/Building-a-Hub-instance. It would be great if you could get it up and running on an updated ubuntu LTS VM. Thank you.
